testing/test_reports.py

```diff
@@ -224,33 +223,31 @@ def test_extended_report_deserialization(self, testdir: Testdir) -> None:
             if rep.failed:
                 assert newrep.longrepr == str(rep.longrepr)
 
-    def test_paths_support(self, testdir: Testdir) -> None:
-        """Report attributes which are py.path or pathlib objects should become strings."""
-        testdir.makepyfile(
+    def test_paths_support(self, pytester: Pytester) -> None:
+        """Report attributes which is pathlib object should become string."""
+        pytester.makepyfile(
             """
             def test_a():
                 assert False
         """
         )
-        reprec = testdir.inline_run()
+        reprec = pytester.inline_run()
         reports = reprec.getreports("pytest_runtest_logreport")
         assert len(reports) == 3
         test_a_call = reports[1]
-        test_a_call.path1 = testdir.tmpdir  # type: ignore[attr-defined]
-        test_a_call.path2 = Path(testdir.tmpdir)  # type: ignore[attr-defined]
+        test_a_call.path = pytester.path
```

Comment:
should I keep check for `py.path.local` as well, for regression?

Reply:
Yes, the test seems to want to test py.path explicitly (in additional to pathlib) so we should keep at, until we get rid of py.path entirely.

Reply:
added back in 5d4284f